src/System.Text.Json/src/System/Text/Json/Serialization/JsonClassInfo.AddProperty.cs

```diff
@@ -11,7 +11,14 @@ internal partial class JsonClassInfo
     {
         private JsonPropertyInfo AddPolicyProperty(Type propertyType, JsonSerializerOptions options)
         {
-            return AddProperty(propertyType, propertyInfo : null, classType : propertyType, options);
+            // A policy property is not a real property on a type; instead it leverages the existing converter
+            // logic and generic support to avoid boxing. It is used with values types and elements from collections and
```

Comment:
Which of the two did you mean here: `... values, types, and elements... ` OR `... value types and elements...`?